inc/cleantalk-public-integrations.php

```diff
@@ -1791,6 +1793,14 @@ function apbct_form__contactForm7__testSpam($spam, $_submission = null)
 
         add_filter('wpcf7_display_message', 'apbct_form__contactForm7__showResponse', 10, 2);
 
+        add_filter('wpcf7_skip_mail', function () {
+            add_filter("wpcf7_feedback_response", function ($response) {
+                $response["status"] = "mail_sent_ng";
+                $response["message"] = "***Forbidden. Sender blacklisted. Blocked by CleanTalk***";
```

Comment:
The block message must be gathered from the cloud response. Use `$ct_result->comment` instead of hard-coded block message.